cortix/__init__.py

```diff
@@ -3,11 +3,11 @@
 # This file is part of the Cortix toolkit environment
 # https://cortix.org
 
-from .src.cortix_main import Cortix
+from .cortix_main import Cortix
 
-from .src.network import Network
-from .src.module import Module
-from .src.port import Port
+from .network import Network
```

Comment:
This will require my `PYTHONPATH` to point to `path/to/cortix/src` correct? Currently my `PHTHONPATH` points to `path/to/cortix`.